tests/test-cases/Group1-Docker-Commands/1-05-Docker-Start.robot

```diff
@@ -120,3 +120,12 @@ Start a container with removed network
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start ${container}
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  network test-network not found
+
+Simple start with attach
+    Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
+    ${rc}  ${container}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create ${busybox} ls
```

Comment:
Since you are collecting a `rc` value here, should you check it before you proceed further?  Just an observation.